README.md

```diff
@@ -1,2 +1,84 @@
-# module-ballerinax-health.dicom
-The Ballerina DICOM package provides functionalities to work with Digital Imaging and Communications in Medicine (DICOM) standard in Ballerina programs.
+# Ballerina DICOM Packages
+
+The Ballerina DICOM packages provide functionalities to work with Digital Imaging and Communications in Medicine (DICOM) standard in Ballerina programs.
+
+These packages include DICOM structures and data types, encoders, validators, parsers, DICOMweb related utils and a DICOM service type for creating DICOMweb APIs, as well as other miscellaneous utilities for working with the DICOM standard.
```

Comment:
Let's mention about each package we are providing and short description and we can link to module.md of each package to point the sample usage from here

Reply:
Added in https://github.com/ballerina-platform/module-ballerinax-health.dicom/commit/3807383514d097fe372c936aa46ef84a627a58d3